homeassistant/config.py

```diff
@@ -208,22 +203,25 @@ def _filter_bad_internal_external_urls(conf: dict) -> dict:
 )
 
 
+def _raise_issue_if_historic_currency(hass: HomeAssistant, currency: str) -> Any:
```

Comment:
Looks like the retval is always None.
```suggestion
def _raise_issue_if_historic_currency(hass: HomeAssistant, currency: str) -> None:
```